variantEffectPredictor.wdl

```diff
@@ -0,0 +1,381 @@
+version 1.0
+workflow variantEffectPredictor {
+  input {
+    File vcfFile
+    File? targetBed
+    String? custom
+    Boolean toMAF
+    Boolean onlyTumor
+    String vepCacheModule
+    String vepCacheDir
+    String pluginModule 
+    String pluginDir
+  }
+
+  String customCommand = if defined(custom) == true then "true" else "false"
+
+  if (defined(targetBed) == true) {
+    call targetBedTask {
+      input: vcfFile = vcfFile, 
+             targetBed = targetBed
+    }
+  }
+  
+  call vep {
+    input: vcfFile =  select_first([targetBedTask.targetedVcf, vcfFile]),
+           customCommand = customCommand,
+           custom = custom,
+           vepCacheModule = vepCacheModule,
+           vepCacheDir = vepCacheDir,
+           pluginModule = pluginModule, 
+           pluginDir = pluginDir
+    }
+  
+  if (toMAF == true) {
+    if (onlyTumor == true) {
+      call tumorOnlyAlign {
+      input: vcfFile = vep.vepVcfOutput,
```

Comment:
The input for this should be input VCF and not the VEP annotated VCF

Reply:
Looks like:
vcf2maf is enabled for tumor only VCFs
and it reads VEP annotated VCF to run vcf2maf